pom.xml

```diff
@@ -266,6 +267,101 @@
 
   <dependencyManagement>
     <dependencies>
+      <dependency>
+        <groupId>org.eclipse.jetty</groupId>
+        <artifactId>jetty-continuation</artifactId>
+        <version>${jetty.version}</version>
+      </dependency>
+      <dependency>
+        <groupId>org.eclipse.jetty</groupId>
+        <artifactId>jetty-http</artifactId>
+        <version>${jetty.version}</version>
+      </dependency>
+      <dependency>
+        <groupId>org.eclipse.jetty</groupId>
+        <artifactId>jetty-io</artifactId>
+        <version>${jetty.version}</version>
+      </dependency>
+      <dependency>
+        <groupId>org.eclipse.jetty</groupId>
+        <artifactId>jetty-jaas</artifactId>
+        <version>${jetty.version}</version>
+        <exclusions>
+          <exclusion>
+            <groupId>*</groupId>
```

Comment:
Why the exclusions?

Reply:
As the only usage of these artifacts had these exclusions, I've simply moved the exclusions to the parent pom.
Is it preferable that I move the exclusions to the poms where the dependency is used?

Reply:
> Is it preferable that I move the exclusions to the poms where the dependency is used?
Yes. Unless there is a good reason to exclude at the top level of all projects the transitives, the exclusions should be at the "leaf" projects/modules. The same applies to the remaining exclusions in this PR.

Reply:
Ok, will do that!
I'll resolve the other conversations for they're the same as this one.

Reply:
Pushed new code for this PR: resolving this conversation.
New conversations may be created when reviewing the new code - I hope not :-)